src/index.tsx

```diff
@@ -1,16 +1,21 @@
 import * as React from 'react'
 import * as ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
-import { store } from './configStore'
+import configureStore, { history } from './configStore'
```

Comment:
Then this could be just `import { configureStore, history}` from './configureStore'`